core/src/mindustry/world/blocks/Autotiler.java

```diff
@@ -112,24 +144,47 @@ default TextureRegion botHalf(TextureRegion input){
         return blendresult;
     }
 
+    /**
+     * Transforms the autotiler setting the connection and the y-scale
+     *
+     * @param num The number to use to transform the array
+     * @param bits The blending value array
+     */
     default void transformCase(int num, int[] bits){
-        if(num == 0){
-            bits[0] = 3;
-        }else if(num == 1){
-            bits[0] = 4;
-        }else if(num == 2){
-            bits[0] = 2;
-        }else if(num == 3){
-            bits[0] = 2;
-            bits[2] = -1;
-        }else if(num == 4){
-            bits[0] = 1;
-            bits[2] = -1;
-        }else if(num == 5){
-            bits[0] = 1;
+        switch (num){
```

Comment:
I don't understand why people think `switch` is somehow less messy. This is *more lines of code*, and you need a `break` statement after every case. It's not any better than an `if`-chain.

Reply:
Really, it just matters as how its implemented throughout the code, and, after checking, this is how its used a lot of the time, so I'll revert it.